configure.ac

```diff
@@ -53,7 +53,7 @@ AC_ARG_WITH(nopylint,
 
 [if test "$with_nopylint" = "no"]
 [then]
-    AC_PATH_PROG([PYLINT], [python3-pylint], [no])
+    AC_PATH_PROG([PYLINT], [pylint-3], [no])
```

Comment:
@rluzynski yes, you are right, AC_PATH_PROGS works like a charm, tested to build on both F25 and F26